invenio_app_ils/config.py

```diff
@@ -829,6 +829,7 @@ def _(x):
     eitems=dict(
         aggs=dict(
             access=dict(terms=dict(field="open_access")),
+            eitemtype=dict(terms=dict(field="eitem_type")),
```

Comment:
for naming consistency: 
```suggestion
            eitem_type=dict(terms=dict(field="eitem_type")),
```